docs/src/doe.md

```diff
@@ -0,0 +1,159 @@
+# DOE Stack
+The Caliptra subsystem supports SPDM, Secure-SPDM over PCI Data Object Exchange (DOE) mailbox protocol. The following diagram gives the over view of the DOE send and receive stack.
+
+DOE Receive stack:
+
+![The DOE Tock receive stack](images/doe_tock_receive.svg)
```

Comment:
This image is hard to read -- could you crop it so there isn't so much space around the content?